components/navigation/ui/src/main/resources/data/WebHome.xml

```diff
@@ -37,7 +37,33 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>false</hidden>
-  <content>{{dashboard style="panels" /}}</content>
+  <content>{{velocity}}
+##Create the dashboard area
+#set ($numColumns = 2)
+$xwiki.ssfx.use('uicomponents/dashboard/dashboard.css', true)##
+$xwiki.ssfx.use('uicomponents/container/columns.css', {'forceSkinAction' : true, 'columns' : $numColumns})##
+(% id="dashboard" class="panels container-columns"%)(((
+  #foreach ($column in [1..$numColumns])
+  (% class="column #if($column == 1)first-column#elseif($column==$numColumns)last-column#end" %)(((
+    #foreach ($e in $services.uix.getExtensions('org.phenotips.homepage.dashboard', {'sortByParameter' : 'priority'}))
+      #set ($extensionParameters = $e.getParameters())
+      #if ($extensionParameters.get("column") == "$column" &amp;&amp; $extensionParameters.get("enabled") != 'false')
+        #set ($extentionContent = $services.rendering.render($e.execute(), 'xhtml/1.0').trim())
+        #set ($gadgetDocRef = $e.getDocumentReference())
+        #set ($gadgetDoc = $xwiki.getDocument($gadgetDocRef))
+        {{html wiki=true clean=false}}
```

Comment:
I know the gadget content is supposed to be fully rendered but setting ```wiki=false``` leads to xwiki markup and ```{{html}}``` macros to appear in plain text. I could go through each gadget's code to see if this can be resolved but I think it might be safer to leave ```wiki=true```. Unless, of course there are strong reasons otherwise.